src/AzureDevOpsPolicyConfigurator/Logic/PolicyExecuterBase.cs

```diff
@@ -219,7 +218,7 @@ public async Task Execute(ExecuterSettings arguments)
                             }
                             else
                             {
-                                resultList.Add(() => this.UpdatePolicy(policyClient, types, projectId, repository, currentPolicy, policy, serverPolicy));
+                                resultList.Add(Task.Run(() => this.UpdatePolicy(policyClient, types, projectId, repository, currentPolicy, policy, serverPolicy)));
```

Comment:
Why calling `Task.Run` here?

Reply:
Did not find a better solution to await void methods:
https://stackoverflow.com/questions/13636648/wait-for-a-void-async-method

Reply:
Found a better way